src/custom/hooks/useSwapCallback.ts

```diff
@@ -88,7 +69,6 @@ async function _postOrder(params: PostOrderParams): Promise<string> {
     appData: DEFAULT_APP_ID, // TODO: Add appData by env var
     feeAmount: '0', // TODO: Get fee
     kind: trade.tradeType === TradeType.EXACT_INPUT ? ORDER_KIND_SELL : ORDER_KIND_BUY,
```

Comment:
Don't we have `enum OrderKind` in `state/orders/actions`

Reply:
not in the version I had. Let's do a PR with the new version and use it